spec/requests/api/v1/passwords/edit_spec.rb

```diff
@@ -3,16 +3,25 @@
 describe 'GET api/v1/users/passwords/edit' do
   subject { get edit_user_password_path, params: }
 
-  let(:user)           { create(:user, password: 'mypass123') }
-  let(:password_token) { user.send(:set_reset_password_token) }
+  let(:user) { create(:user, password: 'mypass123') }
+  # We have to fix the raw token so this doesn't change in the docs
+  let(:raw) { '96BuszWmzDxRqXYzc_Mf' }
+  let(:key) { Devise.token_generator.send(:key_for, 'reset_password_token') }
+  let(:enc) { OpenSSL::HMAC.hexdigest('SHA256', key, raw) }
```

Comment:
My concern is whether this should be something to execute on Actions or locally since it seems to add more complexity to tests. Will this impact developers with other features? Just a thought.